src/AspNet.Security.OAuth.Introspection/OAuthIntrospectionHandler.cs

```diff
@@ -148,7 +149,7 @@ public class OAuthIntrospectionHandler : AuthenticationHandler<OAuthIntrospectio
         protected virtual Task<bool> ValidateAudienceAsync(JObject payload) {
             // If no explicit audience has been configured,
             // skip the default audience validation.
-            if (string.IsNullOrEmpty(Options.Audience)) {
+            if (Options.Audiences == null || Options.Audiences.Count <= 0) {
```

Comment:
Well, `IList.Count` is `int`, so can be negative. :stuck_out_tongue_winking_eye: